src/NextGen/DonationForm/Blocks/DonationFormBlock/Block.php

```diff
@@ -242,6 +236,30 @@ private function enqueueScripts(int $formId)
             'give'
         ))->loadInFooter()->enqueue();
 
+        // load template
+        /** @var FormTemplateRegistrar $formTemplateRegistrar */
+        $formTemplateRegistrar = give(FormTemplateRegistrar::class);
+
+        // silently fail if template is missing for some reason
+        if ($formTemplateRegistrar->hasTemplate($formTemplateId)) {
+            $template = $formTemplateRegistrar->getTemplate($formTemplateId);
+
+            if ($template->css()) {
+                wp_enqueue_style('givewp-form-template-' . $template->getId(), $template->css());
+            }
+
+            if ($template->js()) {
+                wp_enqueue_script(
+                    'givewp-form-template-' . $template->getId(),
+                    $template->js(),
+                    ['give-donation-form-registrars-js'],
+                    false,
```

Comment:
This does seem to prevent defining dependencies for the script, which may or may not be needed.